rest-service/manager_rest/test/endpoints/test_plugins_updates.py

```diff
@@ -154,6 +153,30 @@ def test_raises_while_plugins_updates_are_active(self):
                 '{0}'.format(plugins_update.id)):
             self.client.plugins_update.update_plugins('hello_world')
 
+    def test_doesnt_raise_when_plugins_updates_are_not_active(self):
```

Comment:
not a good name, not clear what it's testing for

Reply:
Split and changed the names